src/Codeception/Lib/Interfaces/ElementLocator.php

```diff
@@ -30,5 +30,5 @@ interface ElementLocator
      * @api
      * @return Crawler|array of interactive elements
      */
-    public function _findElements(array $locator);
+    public function _findElements(array $locator): array|Crawler;
```

Comment:
Parameter should be `mixed`, because we currently need `string|array|WebDriverBy`.

It is better to leave it more flexible.

Can return type be `iterable`?

Reply:
`is_iterable(\Symfony\Component\DomCrawler\Crawler::class)` returns `false`. So i don't think so.

Reply:
```php
var_dump(is_iterable(new \Symfony\Component\DomCrawler\Crawler)); // bool(true)
```

is the implementation of `\Traversable` mandatory here ?

Reply:
I mean, `iterable` or `array|iterable`?

Reply:
i.e. do I just leave it as an `iterable`?

Reply:
I changed it to `iterable`.